README.md

```diff
@@ -1,28 +1,34 @@
-<div style="display: grid; place-items: center;">
-  <h1>Hyperlight</h1>
-  <img src="https://raw.githubusercontent.com/hyperlight-dev/hyperlight/refs/heads/main/docs/assets/hyperlight-logo.png" width="150px" alt="hyperlight logo"/>
-  <p>
-    <strong>Hyperlight is a lightweight Virtual Machine Manager (VMM) designed to be embedded within applications. It enables safe execution of untrusted code within <i>micro virtual machines</i> with very low latency and minimal overhead.
-    </strong>
-  </p>
+<div align="center">
+    <h1>Hyperlight</h1>
+    <img src="https://raw.githubusercontent.com/hyperlight-dev/hyperlight/refs/heads/main/docs/assets/hyperlight-logo.png" width="150px" alt="hyperlight logo"/>
+    <p><strong>Hyperlight is a lightweight Virtual Machine Manager (VMM) designed to be embedded within applications. It enables safe execution of untrusted code within <i>micro virtual machines</i> with very low latency and minimal overhead.</strong></p>    
 </div>
 
-> Note: Hyperlight is a nascent project with an evolving API and no guaranteed support. Assistance is provided on a best-effort basis by the developers.
+> Note: Hyperlight is a nascent project with an evolving API and no guaranteed support. Assistance is provided on a
+> best-effort basis by the developers.
 
 ---
 
 ## Overview
 
-Hyperlight is a library for creating _micro virtual machines_  — or _sandboxes_ — specifically optimized for securely running untrusted code with minimal impact. It supports both Windows and Linux, utilizing [Windows Hypervisor Platform](https://docs.microsoft.com/en-us/virtualization/api/#windows-hypervisor-platform) on Windows, and either Microsoft Hypervisor (mshv) or [KVM](https://linux-kvm.org/page/Main_Page) on Linux.
+Hyperlight is a library for creating _micro virtual machines_  — or _sandboxes_ — specifically optimized for securely
+running untrusted code with minimal impact. It supports both Windows and Linux,
+utilizing [Windows Hypervisor Platform](https://docs.microsoft.com/en-us/virtualization/api/#windows-hypervisor-platform)
+on Windows, and either Microsoft Hypervisor (mshv) or [KVM](https://linux-kvm.org/page/Main_Page) on Linux.
 
-These micro VMs operate without a kernel or operating system, keeping overhead low. Instead, guests are built specifically for Hyperlight using the Hyperlight Guest library, which provides a controlled set of APIs that facilitate interaction between host and guest:
+These micro VMs operate without a kernel or operating system, keeping overhead low. Instead, guests are built
```

Comment:
Are these manual linebreaks intended? I'm no expert on markdown, and I'm not sure if you should linebreak manually or not :D

Reply:
They are not—though, a lot of people add them. My editor has automatic soft-wrapping so it's a non-issue for me 🤷 . Doesn't hurt to leave them, though.

Reply:
Oh, I just realized there were a lot of changes to the README besides what I added. I think this was applied by auto-formatting.